### ISSUE TITLE
PT502 send command

### ISSUE THREAD

**Issue by and12345** [tananaev/traccar#1463]
Hi Anton, 

Just got some documentation on PT502 send command : 

GPRS PROTOCOL
|    Frame header    |    Verify    |    Data body length    |    Data Body    |    

Remark:
Frame header : Start of the frame. Fixed as 0xBF, 0xFB
Verify: XOR of Data Body. 1 byte
Data body length: length of frame, which include “Frame header, Verify, Data body length ,Data Body”, 2 bytes, Little Endian format, ie. Least Significant Byte first
Data body: Format of Command or terminal Reply..

Commonly, the fields, including Frame head, Verify, Data body length, is not necessary; If the Data Body has unrecognizable binary data, in order to parse frame correctly, full frame package is suggested. Binary-date-type frame includes: FOTA, multi-media and etc. Terminal should support both full frame and simplified frame, which has Data Body only.
 Without specification, multi-byte binary data uses Little Endian format, ie. Least Significant Byte first

Stop Car / Cut Engine : OPC1,1
Restore Car / Engine: OPC1,0

Is this can help with PT502 send command ?

**Comment by tananaev** [comment]
Yes, it will definitely help. The problem is lack of time to implement it.

**Comment by and12345** [comment]
Hi Anton, take your time :D. In the meanwhile i'll do what i can to help the development of this great project.
